### PR TITLE
Upgrade PocketIC

### PR DESCRIPTION
**Pull request by frederikrothenberger**
A new PocketIC version has been released. Hence the upgrade.

<hr/><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -35,7 +35,7 @@ ic-response-verification = "2.6"
 ic-stable-structures = "0.6"
 ic-verifiable-credentials = {git = "https://github.com/dfinity/verifiable-credentials-sdk", rev = "b74c746ea5361af3da207a2c957be4a951f7a72c"}
 ic-canister-sig-creation = "1.1"
-pocket-ic = "4.0"
+pocket-ic = "5.0"
```
Comment (lmuntaner) [inline Cargo.toml:38]: This doesn't match the version in downloading?

**Review by lmuntaner — APPROVED** [review]
Approving, but a weird questions

File: Cargo.toml (hunk 1 above)
Comment (frederikrothenberger) [reply Cargo.toml:38]: Client library version and binary are out of sync and have different versions: https://docs.google.com/document/d/1VYmHUTjrgbzRHtsAyRrI5cj-gWGs7ktTnutPvUMJioU/edit#heading=h.5wf28dvt742x

Comment (lmuntaner) [reply Cargo.toml:38]: 🙈

**Pull request merged**